### PR TITLE
Add cvar to limit entities passed into nearby command

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Another day another one of my admins crashed rmc14
See https://github.com/space-wizards/RobustToolbox/pull/5282 for reasoning
Default 5 (you are usually doing self nearby not entities nearby which was the problem)

### CODE REVIEW

File: Robust.Shared/Utility/CollectionExtensions.cs [hunk 1]
```diff
@@ -283,5 +283,24 @@ public static T Pop<T>(this IList<T> list)
             value = default;
             return false;
         }
+
+        public static bool HasMoreThan<T>(this IEnumerable<T> enumerable, int than)
+        {
+            if (enumerable.TryGetNonEnumeratedCount(out var count))
+                return count > than;
+
+            using var enumerator = enumerable.GetEnumerator();
```
Comment (PJB3005) [inline Robust.Shared/Utility/CollectionExtensions.cs]: Does toolshed's architecture guarantee that double enumeration like this won't cause a problem?

Comment (DrSmugleaf) [reply Robust.Shared/Utility/CollectionExtensions.cs]: I don't think so, in testing it worked fine with self and nearby but I don't think there are any guarantees

Comment (DrSmugleaf) [reply Robust.Shared/Utility/CollectionExtensions.cs:292]: The other option is doing ToArray at the start and hoping that there are not hundreds of thousands of entities

Comment (DrSmugleaf) [reply Robust.Shared/Utility/CollectionExtensions.cs]: Fixed in [`6a420ae` (#5355)](https://github.com/space-wizards/RobustToolbox/pull/5355/commits/6a420ae6c887afd126a0655c57a6333d3052c153)

**Pull request merged**